app/routes/project.tsx

```diff
@@ -41,6 +41,22 @@ export default function Index() {
       restoreLocalScopes();
     }
   };
+
+  const newDependencyButton = () => {
+    return (
```

Comment:
follow component nameing rules, and maybe shore syntax

```suggestion
  const NewDependencyButton = () => (<Button...);
```